From 29a70ab34ea3112b8afc8e52cfe51b10b80b32ae Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:38:35 -0400
Subject: [ethernet] i40e/i40evf: remove chatty reset messages

Message-id: <1409658042-22689-155-git-send-email-sassmann@redhat.com>
Patchwork-id: 90350
O-Subject: [RHEL7.1 PATCH 154/281] i40e/i40evf: remove chatty reset messages
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

Both the PF side and the VF side of the VF reset process are too noisy.
We already warn the user that a reset is happening, and that is
sufficient.

Because some of these message are inside if statements, we have to
rejigger the brackets at the same time to keep our coding style
consistent.

Change-ID: Id175562fb0ec7c396d9de156b4890e136f52d5f4
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 63158f916f21b986743d0cb6f9e273d54a01ce7e)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index 71b2f3f..83d90bc 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -5623,8 +5623,6 @@ static void i40e_reset_and_rebuild(struct i40e_pf *pf, bool reinit)
  /* tell the firmware that we're starting */
  i40e_send_version(pf);
 
- dev_info(&pf->pdev->dev, "reset complete\n");
-
 end_core_reset:
  clear_bit(__I40E_RESET_RECOVERY_PENDING, &pf->state);
 }
-- 
1.7.1